functions/helpers/firebaseMock.js

```diff
@@ -0,0 +1,29 @@
+const firebasemock = require("firebase-mock");
```

Comment:
What's the reason you're using firebase-mock? Does this try tests against a sandbox environment?